src/nyx/environment.h

```diff
@@ -117,12 +117,12 @@ class Environment: public BasicEnvironment
 
 	bool singleROI = false; // is set to 'true' parse_cmdline() if labels_dir==intensity_dir
 
-#ifdef USE_ARROW
 
+	bool use_arrow = false;

```

Comment:
"use_arrow" will be defined as a class member no matter if we have Apache functionality enabled or disabled. Can we have no reference to Arrow if USE_ARROW is not defined ?